src/dotnet-roslyn-tools/CreateReleaseTags/AbstractReleaseTagger.cs

```diff
@@ -53,21 +54,53 @@ public async Task CreateReleaseTagsAsync(
                 continue;
             }
 
-            logger.LogInformation("Tagging {CommitSha} as '{TagName}'.", build.CommitSha, tagName);
+            if (TryCreateTag(release, build, tagName))
+            {
+                tagsAdded++;
+            }
+            else
+            {
+                await TryTagFromVmrBuildAsync(release, build, tagName);
```

Comment:
Instead of trying to query both, it looks like the AzDo api exposes the build repository - https://learn.microsoft.com/en-us/rest/api/azure/devops/build/builds/get?view=azure-devops-rest-7.1#build

Could we instead check that to determine if we need to specially handle the vmr?

Reply:
i was confused - `build` is not an azdo build, but a tag from the SDK